src/main/java/org/saltyrtc/ws/test/Test.java

```diff
@@ -0,0 +1,84 @@
+package org.saltyrtc.ws.test;
+
+import com.neovisionaries.ws.client.DualStackMode;
+import com.neovisionaries.ws.client.ProxySettings;
+import com.neovisionaries.ws.client.WebSocket;
+import com.neovisionaries.ws.client.WebSocketAdapter;
+import com.neovisionaries.ws.client.WebSocketException;
+import com.neovisionaries.ws.client.WebSocketFactory;
+import com.neovisionaries.ws.client.WebSocketFrame;
+import com.neovisionaries.ws.client.WebSocketState;
+
+import java.net.URI;
+import java.util.List;
+import java.util.Map;
+
+public class Test {
+    public static void main(String[] args) throws Exception {
+        WebSocketFactory factory = new WebSocketFactory();
+
+        // Apply settings
+        URI uri = new URI("wss://server.saltyrtc.org/debc3a6c9a630f27eae6bc3fd962925bdeb63844c09103f609bf7082bc383610");
+        int timeout = 10000;
+//        ProxySettings proxySettings = factory.getProxySettings();
+//        proxySettings.setServer("https://proxy.saltyrtc.org:3128");
+        factory.setDualStackFallbackDelay(1337);
+//        factory.setDualStackMode(DualStackMode.IPV6_ONLY);
```

Comment:
Dead code? Or do you want to leave this here as a hint for doing manual tests?

Reply:
Err... I think this file was just for testing purposes and is not going to be made upstream.